File: features/fintech/ovo_pay_with_qr/src/main/java/com/tokopedia/ovo/model/WalletData.java
package com.tokopedia.ovo.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WalletData {
    @SerializedName("wallet")
    @Expose
    private Wallet wallet;

    public Wallet getWallet(){
        return wallet;
    }
}

File: features/discovery/home/src/main/java/com/tokopedia/v2/home/model/pojo/wallet/WalletData.kt
package com.tokopedia.v2.home.model.pojo.wallet

import com.google.gson.annotations.SerializedName

data class WalletData(
        @SerializedName("wallet")
        val wallet: Wallet = Wallet(),
        @SerializedName("tokopointsDrawer")
        val tokopoint: Tokopoint = Tokopoint()
) {
    override fun equals(other: Any?): Boolean {
        if (this === other) return true
        if (javaClass != other?.javaClass) return false

        other as WalletData

        if (wallet != other.wallet) return false
        if (tokopoint != other.tokopoint) return false

        return true
    }

    override fun hashCode(): Int {
        var result = wallet.hashCode()
        result = 31 * result + tokopoint.hashCode()
        return result
    }
}